www/about/policies/terms-of-service.md

```diff
@@ -0,0 +1,414 @@
+# Gratipay Terms of Service
+
+These Terms of Service ("Terms") are an agreement between you, a user
+("Participant") of the Gratipay service (the "Service"), and Gratipay, LLC
+("Gratipay"). They govern your use of the Service and your relationship with
+Gratipay. When you register as a Gratipay Participant, you agree to be bound by
+these Terms.
+
+If you have any questions about these terms, please contact us:
+
+Gratipay, LLC
+716 Park Road
+Ambridge, PA 15003
+USA
+legal@gratipay.com  
+
+1. Registration, Security, and Privacy
+
+  1. When you register for the Service, you agree to provide Gratipay with
+     complete and accurate registration information. You are responsible for
+keeping your account information, including your contact and payment
+information, up to date. Gratipay takes our Participants' privacy seriously. We
+will only use or share your information and Content as described in our
+[Privacy Policy](/about/policies/privacy), which is incorporated into these
+Terms.
+
+  1. You are responsible for maintaining the security of your account
+     information, including your password. You agree to notify Gratipay
+immediately of any actual or suspected loss, theft, or unauthorized use of your
+password. Gratipay shall not be liable for any loss or damage arising from your
+failure to keep your account secure.
+
+  1. You may only use the Service if you are 13 years old or older, and you
+     represent to us that you are 13 or older. If you are under the age of 18,
+your legal guardian must consent to your use and must agree to be bound by
+these Terms.
+
+1. The Gratipay Model and General Rules
+
+  1. The Service is a platform to enable Teams of Gratipay Participants to
+     receive payments to fund Open Work. Open Work means that the Team provides
+a clear path for any individual to voluntarily begin contributing to the Team's
+work and to be paid for their contribution. Some examples of Teams performing
```

Comment:
Over on #3390 we had some discussion about "paid for their contribution." The concern was that projects with little to no revenue can barely make ends meet let alone promise to pay others. The revision we [arrived](https://github.com/gratipay/gratipay.com/issues/3390#issuecomment-100681742) at, applied here, would be something like this:

> to voluntarily begin contributing to the Team's work and to ~~be paid for their contribution~~**share in any revenue the Team generates**.